src/webhook-handler/handler.js

```diff
@@ -23,6 +23,7 @@ module.exports.handleWebhookEvent = (event, context, callback) => {
       if (e instanceof HTTPError) {
         response = errorResponse(e.status, e.message)
       } else {
+        console.log(e)
```

Comment:
Do we need to log the error here

Reply:
This should be fixed in https://github.com/lulibrary/alma-webhook-handler/pull/12/commits/55d6ca7444e0a04e2c564d2deac704b93cefbaab